### PR TITLE
New package: gum-0.4.0

### PR DESCRIPTION
**Pull request by akhiljalagam**
#### Testing the changes
- I tested the changes in this PR: **YES**

#### New package
- This new package conforms to the [quality requirements](https://github.com/void-linux/void-packages/blob/master/Manual.md#quality-requirements): **YES**

### CODE REVIEW

**Comment by akhiljalagam** [comment]
How to get man page and shell completions from go package?

**Comment by tranzystorek-io** [comment]
@akhiljalagam gum generates hidden subcommands `gum man` and `gum completion` that you can run wth `vtargetrun` in `post_build`

File: srcpkgs/gum/template [hunk 1]
```diff
@@ -0,0 +1,17 @@
+# Template file for 'gum'
+pkgname=gum
+version=0.2.0
+revision=1
+build_style=go
+go_import_path="github.com/charmbracelet/gum"
```
Comment (tranzystorek-io) [inline srcpkgs/gum/template:6]:
```suggestion
go_import_path="github.com/charmbracelet/gum"
go_ldflags="-X main.Version=v${version}"
```

**Comment by tranzystorek-io** [comment]
Please squash these changes into the initial commit

File: srcpkgs/gum/template [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+
+post_install() {
+	vlicense LICENSE
+
+	for shell in $_completions; do
+		vcompletion ${pkgname}.${shell} $shell gum
+	done
+
+	vman ${pkgname}.1
+}
```
Comment (classabbyamp) [inline srcpkgs/gum/template:35]:
```suggestion
post_install() {
	vlicense LICENSE

	_cli=$(find $GOPATH/bin -name $pkgname)
	for shell in bash fish zsh; do
		vtargetrun $_cli completion $shell > ${pkgname}.${shell}
		vcompletion ${pkgname}.${shell} $shell
	done

	vtargetrun $_cli man > ${pkgname}.1
	vman ${pkgname}.1
}
```
this is a bit neater

**Comment by dmarto** [comment]
gum has newly released v0.4.0, please update

**Comment by classabbyamp** [comment]
commit message needs to be updated, and the commit body can be removed

**Comment by akhiljalagam** [comment]
> commit message needs to be updated, and the commit body can be removed

done

**Pull request closed without merging**